runtime/astar/src/weights/pallet_xcm.rs

```diff
@@ -272,4 +277,12 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(9_u64))
 			.saturating_add(T::DbWeight::get().writes(4_u64))
 	}
+
+	fn new_query() -> Weight {
+		todo!()
+	}
+
+	fn take_response() -> Weight {
+		todo!()
+	}
```

Comment:
This would panic 😓 , it better if we just return `weight::MAX` here

same comment for other occurrences

Reply:
This is temporary until I run the benchmarks